flocker/node/_loop.py

```diff
@@ -597,8 +604,16 @@ def error(failure):
         d.addErrback(error)
 
         # We're done with the iteration:
-        d.addCallback(
-            lambda delay: self.fsm.receive(delay))
+        def send_delay_to_fsm(delay):
+            Message.log(
+                message_type=u'flocker:node:_loop:CONVERGE:delay',
+                log_level=u'INFO',
+                message=u'Delaying until next convergence loop.',
+                delay=delay
+            )
+            return self.fsm.receive(delay)
```

Comment:
I do see other places where the loop delay is logged e.g.

```
{"_PROCESS_NAME": "flocker-container-agent", "fsm_identifier": "<flocker.node._loop.ConvergenceLoop object at 0x7f337c02bb50>", "fsm_input": "<ConvergenceLoopInputs=SLEEP>", "timestamp": 1457971496.563045, "fsm_rich_input": "<_Sleep>", "action_status": "started", "task_uuid": "fb0e1146-2ba3-4766-9b5f-3e44f86fec95", "_HOSTNAME": "104.197.90.163", "action_type": "fsm:transition", "fsm_state": "<ConvergenceLoopStates=CONVERGING>", "task_level": [2, 2, 2, 2, 2, 2, 7, 1]}
{"_PROCESS_NAME": "flocker-container-agent", "fsm_next_state": "<ConvergenceLoopStates=SLEEPING>", "timestamp": 1457971496.565061, "action_status": "succeeded", "task_uuid": "fb0e1146-2ba3-4766-9b5f-3e44f86fec95", "_HOSTNAME": "104.197.90.163", "action_type": "fsm:transition", "fsm_output": ["<ConvergenceLoopOutputs=SCHEDULE_WAKEUP>"], "task_level": [2, 2, 2, 2, 2, 2, 7, 2]}

```

Do you think this new / extra logging would have helped avoid the confusion in reading the acceptance test logs?

Personally, I think I might have skipped over both...in search of dataset / volume operations.